README.md

```diff
@@ -1 +1,63 @@
-# Porfolio
+![Microverse](https://img.shields.io/badge/Microverse-blueviolet)
+
+## Portfolio
+
+
+![](https://img.shields.io/github/followers/sheylaPozo?style=social)
+
+# Module 1: "Portfolio" project #3 of Microverse 2.0
+
+> This project is part of Microverse's The Portal part of the curriculum. and it's been built with Html 5 and CSS, as part of Microverse 2.0 curriculum.
+
+## 💖 Built With 
+
+---
+
+- HTML
+- CSS
+
+---
+
+*💛 Image Preview 💛*
+
+![Demo](https://user-images.githubusercontent.com/54015740/121461687-f3e20f80-c974-11eb-9cb7-f573e0987ad4.jpg)
+
+---
+The main purpose of the project [Portfolio](https://sheylapozo.github.io/Porfolio/) was for me to develop some hands on work using my previous knokledge working with Html, CSS, and linters, which also serves as practice in documenting my work in a professional way, with the learning objectives of using the best practices in HTML code, semantic HTML tags, CSS selectors in a correct way, CSS box model, to understand how to parse a Figma design to create a UI., learning how to use Flexbox to place elements in the page, learning how to build a personal portfolio site, as well as how to use images and backgrounds to enhance the look of the website which enhance the experience in programming with HTML and CSS. 
```

Comment:
- [ ] Kindly make sure that you have no misspellings or grammar mistakes with your written text in the README files or the PR summaries.
Please notice that such a small mistake might leave a bad impression on future viewers.
I know that this stuff happens to all of us all the time without noticing but, we're here for you 🤗
I recommend using [Grammarly](https://app.grammarly.com/) to correct your texts as you type them.

Reply:
done thanks